admin-manual/installation-setup/installation/install-ubuntu.rst

```diff
@@ -0,0 +1,425 @@
+
+   Run these three commands when the final release packages are made:
+
+   .. code:: bash
+
+      sudo wget -O - https://packages.archivematica.org/1.7.x/key.asc  | sudo apt-key add -
+      sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.7.x/ubuntu xenial main" >> /etc/apt/sources.list'
+      sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.7.x/ubuntu-externals xenial main" >> /etc/apt/sources.list'
+
+2. Add Elasticsearch package source (optional). Elasticsearch comes from its own
+   package repository.
+
+   .. note:: Skip this step if you are planning to run Archivematica in indexless
+      mode (without Elasticsearch).
+
+   .. code:: bash
+
+      sudo wget -O - http://packages.elasticsearch.org/GPG-KEY-elasticsearch | sudo apt-key add -
+      sudo sh -c 'echo "deb http://packages.elasticsearch.org/elasticsearch/1.7/debian stable main" >> /etc/apt/sources.list'
+
+   At this point you will need to restart dashboard and mcp-server services
+
+   .. code:: bash
+
+      sudo service archivematica-dashboard restart
+      sudo service archivematica-mcp-server restart
+
+
+3. Update to the most recent OS release - for Xenial, this is 16.04.4. This step
+   will also fetch a list of the software from the package repositories you just
+   added to your system.
+
+   .. code:: bash
+
+      sudo apt-get update
+      sudo apt-get upgrade
+
+4. Install Elasticsearch (optional)
+
+   .. note:: Skip this step if you are planning to run Archivematica in indexless
+      mode (without Elasticsearch).
+
+   .. code:: bash
+
+      sudo apt-get install elasticsearch
+
+5. Install the Storage Service package.
+
+   .. code:: bash
+
+      sudo apt-get install -y archivematica-storage-service
+
+6. Configure the Storage Service.
+
+   .. code:: bash
+
+      sudo rm -f /etc/nginx/sites-enabled/default
+      sudo ln -s /etc/nginx/sites-available/storage /etc/nginx/sites-enabled/storage
+
+   .. warning:: If you are planning to use the `Sword API`_ of the Archivematica
+      Storage Service, then (due to a `known issue`_), you must instruct
+      Gunicorn to use the ``sync`` worker class:
+
+   .. code:: bash
+
+      sudo sh -c 'echo "SS_GUNICORN_WORKER_CLASS=sync" >> /etc/default/archivematica-storage-service'
+
+7. Update pip. This is used to install Python dependencies for both the Storage
+   Service and the dashboard. There is a `known issue with pip`_ on Ubuntu 14.04
+   that makes this step necessary. This step is optional on Ubuntu 16.04, but is
+   still a good idea to get the most recent version of pip.
+
+   .. code:: bash
+
+      sudo wget https://bootstrap.pypa.io/get-pip.py
+      sudo python get-pip.py
+
+8. Install the Archivematica packages. The order of installation is important -
+   the mcp-server package must be installed before the dashboard package. While
+   it is possible to install the mcp-client package on a separate machine, that
```

Comment:
`s/mcp-client/archivematica-mcp-client`

^ that's the formal name of the package after all.